server/views/serviceProviderReferrals/interventionProgress.njk

```diff
@@ -14,6 +14,10 @@
      {{ govukNotificationBanner(cancelledReferralNotificationBannerArgs) }}
   {% endif %}
 
+  {% if presenter.referralAssigned %}
+    <p class="govuk-body">This intervention is assigned to <strong>{{ presenter.assignedCaseworkerFullName }}</strong> (<a href="mailto:{{ presenter.assignedCaseworkerEmail }}">{{ presenter.assignedCaseworkerEmail }}</a>).</p>
+  {% endif %}
```

Comment:
we don't need the text: `This intervention is not yet assigned to a caseworker` ?